packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -71,79 +72,71 @@ class DayViewBase extends React.PureComponent<VerticalViewProps, ViewState> {
     timeTableRowComponent: 'TimeTableRow',
   };
 
-  viewCellsDataBaseComputed = ({
-    currentDate,
-  }) => {
-    const { intervalCount, startDayHour, endDayHour, cellDuration } = this.props;
-    return viewCellsDataCore(
-      currentDate, undefined,
-      intervalCount, [],
-      startDayHour!, endDayHour!, cellDuration!,
-      Date.now(),
-    );
-  }
-
   timeTableElementComputed = () => this.timeTable;
   layoutElementComputed = () => this.layout;
   layoutHeaderElementComputed = () => this.layoutHeader;
 
-  layoutHeaderElement: ComputedFn = (getters) => {
-    const { name: viewName } = this.props;
+  layoutHeaderElement = memoize(viewName => (getters) => {
     return computed(
       getters, viewName!, this.layoutHeaderElementComputed, getters.layoutHeaderElement,
     );
-  }
+  });
 
-  layoutElement: ComputedFn = (getters) => {
-    const { name: viewName } = this.props;
+  layoutElement = memoize(viewName => (getters) => {
     return computed(
       getters, viewName!, this.layoutElementComputed, getters.layoutElement,
     );
-  }
+  });
 
-  timeTableElement: ComputedFn = (getters) => {
-    const { name: viewName } = this.props;
+  timeTableElement = memoize(viewName => (getters) => {
     return computed(
       getters, viewName!, this.timeTableElementComputed, getters.timeTableElement,
     );
-  }
+  });
 
-  viewCellsDataComputed: ComputedFn = (getters) => {
-    const { name: viewName } = this.props;
-    return computed(
-      getters, viewName!, this.viewCellsDataBaseComputed, getters.viewCellsData,
+  viewCellsDataBaseComputed = (startDayHour, endDayHour, cellDuration) => ({
```

Comment:
This function should be a method not an instance function (because it is called with proper context).